### ISSUE TITLE
Is that true the source and target embedding space should have the same dimensionality?

### ISSUE THREAD

**Issue by yuchenlin** [artetxem/vecmap#2]
Hi Mikel,

I found that you didn't mention the case that the source embeddings and target embeddings can have different dimensionality (for example, english 300d, and Chinese 100d). 
Did I miss something in the paper or the code? Or you only focus on the situation that the two embedding spaces must have the same dimensionality? 
 

Thanks and regards,
Bill

**Comment by artetxem** [comment]
You are right, we currently do not support both spaces having different dimensionality. It should not be difficult to implement it, but I do not think that it would be particularly useful either. I will leave the issue open for now.

**Comment by shizhediao** [comment]
what if the source embedding is 200000*300 while the target embedding is 199999*300, could it be trained correctly by this code?
thanks!

**Comment by shizhediao** [comment]
@artetxem

**Comment by artetxem** [comment]
@shizhediao that should work ok.
